packages/fx-core/src/plugins/resource/bot/configs/scaffoldConfig.ts

```diff
@@ -64,7 +64,7 @@ export class ScaffoldConfig {
       // convert HostTypeTrigger question to trigger name
       this.triggers = rawHostTypeTriggers
         .map((hostTypeTrigger) => {
-          const option = HostTypeTriggerOptions.find((option) => option.id === hostTypeTrigger);
+          const option = FunctionsOptionItems.find((option) => option.id === hostTypeTrigger);
           return option?.trigger;
         })
         .filter((item): item is NotificationTrigger => item !== undefined);
```

Comment:
Since we restrict HostTypeTriggerOptions to FunctionsOptionItems which is always a NotificationTrigger, do we still need L79 and NotificationTrigger interface anymore?

When we do the refactor, we could take a look at it further to check if there is anything out-of-date and can be removed.